managed/devops/bin/yb_platform_util.py

```diff
@@ -0,0 +1,700 @@
+        # add the missing fields in the config json.
+        universe_config_json['clusterOperation'] = 'CREATE'
+        universe_config_json['currentClusterType'] = 'PRIMARY'
+        # Call universe config api to get the update config data.
+        universe_config_json = self.__post_universe_config(universe_config_json)
+        if universe_config_json:
+            task_id = self.__create_universe_from_config(universe_config_json)
+            print('Universe create requested successfully.')
+            print('Use {0} as task id to get status of universe.'.format(task_id))
+            if not self.args.no_wait:
+                self.__task_progress_bar(task_id)
+        else:
+            print('Unable to Create Universe for Customer {0}'.format(customer_uuid))
+
+
+    def get_single_customer_uuid(self):
+        """
+        Get customer UUID. Raise error in case multiple customer is present. 
+
+        :return: None
+        """
+        customer_url = '{0}/api/v1/customers'.format(self.base_url)
+        response = self.__call_api(customer_url)
+        data = self.__convert_unicode_json(json.load(response))
+        if (len(data) == 1):
+            self.customer_uuid = str(data[0])
+        else:
+            sys.stderr.write('Muliple customer UUID present, Please provide customer UUID')
+            exit()
+
+
+    def get_regions_data(self):
+        """
+        Delete the universe by providing UUID.
+
+        :return: All available regions
+        """
+        provider_url = '{0}/api/v1/customers/{1}/regions'.format(self.base_url, self.customer_uuid)
+        response = self.__call_api(provider_url)
+        region_data = self.__convert_unicode_json(json.loads(response.read()))
+        regions = []
+        for each in region_data:
+            zones = []
+            for each_zone in each.get('zones'):
+                zone = {}
+                zone['name'] = each_zone.get('name')
+                zone['uuid'] = each_zone.get('uuid')
+                zones.append(zone)
+            region = {}
+            region['zones'] = zones
+            region['region_name'] = each.get('name')
+            region['uuid'] = each.get('uuid')
+            region['provider'] = each.get('provider').get('code')
+            regions.append(region)
+        print(json.dumps(regions))
+    
+
+    def get_provider_data(self):
+        """
+        Delete the universe by providing UUID.
+
+        :return: All available providers
+        """
+        provider_url = '{0}/api/v1/customers/{1}/providers'.format(self.base_url, self.customer_uuid)
+        response = self.__call_api(provider_url)
+        provider_data = self.__convert_unicode_json(json.loads(response.read()))
+        providers = []
+        for each in provider_data:
+            provider = {}
+            provider['name'] = each.get('name')
+            provider['uuid'] = each.get('uuid')
+            providers.append(provider)
```

Comment:
better if you directly append instead of using variable
`providers.append({....})`